cmd/root.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright © 2022 SUSE LLC
+1
```

Comment:
What is this `1` here?

Also, do we really need this big license blob in every file? I don't see it in the Rancher Desktop Typescript sources, so why is this needed in the Go files?

Reply:
Not sure how did that `1` got in there, probably from where I copied it from(now I need to figure out where I copied it from 😂 ). General practice for open-source software is to include the header license in all the files. We should probably make that decision to either include everywhere or what is the plan. Because I have seen the random appearance of license in file headers in `rancher-desktop`.

Reply:
Yes, we're _supposed_ to have those for Rancher Desktop TypeScript (and everything else) files too; we just haven't been doing it properly…